src/ApiCaller.js

```diff
@@ -30,10 +30,7 @@ class ApiCaller {
       'Network request failed'
     ];
 
-    let fetchOptions = {
-      method: config.method,
-      headers: config.headers
-    };
+    let fetchOptions = Object.assign({}, config);
```

Comment:
This will pass entire config object to fetch. Let's instead add cors and credentials in the same way as it was earlier?